apollo-backend/src/main/java/io/logz/apollo/controllers/BlockerDefinitionController.java

```diff
@@ -106,6 +107,15 @@ public void addBlockerDefinition(String name, Integer environmentId, Integer ser
                 logger.error("Could not initiate a SingleRegionBlocker with environment or stack");
                 assignJsonResponseToReq(req, HttpStatus.BAD_REQUEST, "SingleRegionBlocker cannot be defined for a specific environment or stack");
                 return;
+            } else if (blockerJsonConfiguration.equals("{}")) {
```

Comment:
Can there be a situation where it is not `"{}"`? What will happen in this case? If it can't happen now, maybe someone will change it in the future? maybe better to handle such a possibility.

Reply:
Can not be a situation where it is not "{}". I added a null check for any case.